recipes/liac-arff/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "liac-arff" %}
+{% set version = "2.5.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3220d0af6487c5aa71b47579be7ad1d94f3849ff1e224af3bf05ad49a0b5c4da
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - ez_setup
```

Comment:
I'm not completely sure about this requirement but it was suggested by Grayskull.

The setup.py file first imports setuptools and then falls back to ez_setup.
See: https://github.com/renatopp/liac-arff/blob/35816f5c393a6a04ed6fcc2d41b0309433fa4cd3/setup.py#L10-L17

Reply:
I'd be inclined to drop it:
```suggestion
```